Docs/macos_docker_sync/mac.md

```diff
@@ -0,0 +1,23 @@
+# Development on a Mac using docker-sync
+Due performance issues while developing on a Mac with docker-compose it’s necessary to use docker-sync.
+
+## Requirements
+Install [docker-sync](http://docker-sync.io/):
+```bash
+gem install docker-sync
+```
+
+Install [unison](https://www.cis.upenn.edu/~bcpierce/unison/):
+```bash
+brew install unison
+```
+
+## File locations
+Just like the compose file template copy `docker-sync.yml.dist` and `docker-compose.mac.yml.dist` to the root of your project or next to you `docker-compose.yml` if you placed it somewhere else. In this case, you have to change the `src:` parameter of you configution. Also don't forget to get rid of the `.dist` file extension for both files. ;)
```

Comment:
configution => configuration 😃

Reply:
Please precise the file in which you eventually have to change the `src:` parameter.